backoffice/src/controllers/auth/index.js

```diff
@@ -0,0 +1,67 @@
+import axios from 'axios';
+
+/**
+ * Call yakchat auth server
+ */
+export function login (username, password) {
+	axios.post('http://localhost:8080/api/api/v1/auth/login', 
+		JSON.stringify({ username, password }), 
+		{headers: {
+			'Accept': 'application/json',
+			'Content-Type': 'application/json'
+		}}
+	)
+	.then((response) => {
+		console.log(response);
+		localStorage.setItem('udata', JSON.stringify({
+			accessLevel: 5,
+			admin: false,
+			uid: '0FxY4aLtrnhRaIzQAbmNNzyarN72',
+			displayName: 'GithubClient',
+			email: 'gh-client-t01@rebeldemo.com',
+			emailVerified: true
+		}))
+		if (process.env.ENVIROMENT === 'PRODUCTION') {
+			document.location.pathname = '/backoffice.html';
+		} else {
+			document.location.pathname = '/'
+		}
+		localStorage.setItem('authorization', response.headers.authorization);
+	})
+	.catch((error) => {
+		console.log(error);
+		global.storage.dispatch({
+			type: 'LOGIN-FAIL',
+			error
+		});
+	})
+}
+
+/**
+ * Call yakchat auth server
+ */
+export function signup (username, password, type, domain, displayName) {
+	axios.post('http://localhost:8080/api/api/v1/auth/signup', 
```

Comment:
btw ... does the endpoint works under `/api/api` ? Please confirm to make a fix on the router ...

Reply:
yes, in the dev server add a proxy pointing localhost:8080/api/ to localhost:8888